Documentation/connectors-configuration.md

```diff
@@ -167,6 +167,10 @@ In addition to `id` and `type`, the `ldap` connector takes the following additio
 
 * skipCertVerification: a `boolean`. Skip server certificate chain verification.
 
+* initConn: a `integer`. Number of LDAP Connections to establish on startup. Default: `5`
```

Comment:
I don't think this is necessary. Let's just let the pool fill as various connections are made.